Policies/github-privacy-statement.md

```diff
@@ -33,13 +34,13 @@ To see our Privacy Notice to residents of California, please go to [GitHub's Not
 
 | Section | What can you find there? |
 |---|---|
-| [Who is responsible for the processing of your information](#who-is-responsible-for-the-processing-of-your-information) | Subject to limited exceptions, GitHub is the controller and entity responsible for the processing of your Personal Data in connection with the Website or Service. |
```

Comment:
Is the change from "Personal Data" to "personal data" a stylistic change?

I note that the paragraph above is still intact:

> All capitalized terms have their definition in [GitHub’s Terms of Service](/github/site-policy/github-terms-of-service), unless otherwise noted here.

Presuming this capitalization change is unintentional, it has the unfortunate effect of decoupling "Personal Data" from the definition provided in the GitHub Terms of Service, which means that "personal data" is no longer as delineated there, but could well be anything.

If this is an intentional change, it would seem better made as a visible change to the Terms of Service. If the intent is not to change the Terms of Service but to arbitrarily expand "personal data" without drawing attention, well, that seems evil.

Reply:
Looking into this further -- it looks like "Personal Data" is defined these days in the [GitHub Data Protection Agreement](https://docs.github.com/en/site-policy/privacy-policies/github-data-protection-agreement). Perhaps this was being decapitalized since it is not directly defined (afaict) in the GitHub Terms of Service?

Reply:
![image](https://user-images.githubusercontent.com/102749793/182579458-eb36124e-79ab-4877-a461-796e65da14b1.png)

Reply:
Oh bet

Reply:
The collection of information and sale of it I think is something that has been going on for a long time. I think what matters is knowing what information we provide. But it's always good to know

Reply:
salve cade os BR

Reply:
in a court of law, doesn't "Personal Data" mean "personal data" ? 

lol

Reply:
555

Reply:
555

Reply:
555

Reply:
555